plugins/lando-events/lib/utils.js

```diff
@@ -41,9 +41,10 @@ exports.events2Runz = (cmds, app, data = {}) => _.map(cmds, cmd => {
   if (app.services && !_.includes(app.services, service)) {
     throw new Error(`This app has no service called ${service}`);
   }
+  const separator = app._config.composeSeparator;
```

Comment:
I know this is not an ideal solution, but I can't seem to find another way to get Lando config when I have only App :-(